data/technology/appliances.json

```diff
@@ -63,6 +65,7 @@
     "oven",
     "panini sandwich grill",
     "patio heater",
+    "pneumatic vacuum".
```

Comment:
@whitten Please can you change that full-stop for a comma? 

I also agree "hvac" should be "HVAC".